agent/installer/bundle_builder/ingredients/deb/Dockerfile

```diff
@@ -12,8 +12,8 @@ ARG BASE_IMAGE=ubuntu:18.04
 FROM $BASE_IMAGE as build
 
 # Override to download other version
-ENV CONTAINERD_VERSION=1.5.7
-ENV KUBERNETES_VERSION=1.21.2-00
+ENV CONTAINERD_VERSION=1.6.0
+ENV KUBERNETES_VERSION=1.23.4-00
```

Comment:
Why there is a need to use `00` at the end of the version? I remember this is required for apt-get to fetch the correct version. but is it required for Docker to pull the image?

Reply:
These are not for pulling images. If you see the [download.sh](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/agent/installer/bundle_builder/ingredients/deb/download.sh) file, these variables are used to download the deb packages via `apt-get`